build-tools/packages/build-infrastructure/src/config.ts

```diff
@@ -0,0 +1,224 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { cosmiconfigSync } from "cosmiconfig";
+
+import {
+	type IPackage,
+	type PackageName,
+	type ReleaseGroupName,
+	isIPackage,
+} from "./types.js";
+
+/**
+ * The version of the fluidRepo configuration currently used.
+ *
+ * @remarks
+ *
+ * This is not exported outside of the build-infrastructure package; it is only used internally.
```

Comment:
These remarks are not correct. This has to be exported because it's used in the definition of the config interface itself, which is definitely exported. Maybe the version should just be hardcoded in the config type; then the constant doesn't need to be exported.